Scripts/script-CommonServer.yml

```diff
@@ -206,7 +207,12 @@ script: |-
           t.forEach(function(entry){
               var vals = [];
               headers.forEach(function(h){
-                    vals.push((!!entry[h] || entry[h]===0) ? stringEscapeMD(formatCell(entry[h], cellDelimiter), true) : ' ');
+                  if(entry[h] === null || entry[h] === undefined){
+                      vals.push(' ');
+                  }
+                  else {
```

Comment:
else should be in the same line of the closing (line 212).
Also, space should be added after the if and before the {